std_msgs/Float32MultiArray orientation_seq
int32 computation_time_steps